build.html

```diff
@@ -22,7 +22,7 @@
 {{/if}}
 
 <!-- MENU -->
-<div class="fl-menu-body fl-menu-circle-nav-list-holder {{#if location}}fl-menu-circle-left{{/if}}">
+<div class="fl-menu-body fl-menu-circle-nav-list-holder {{#if location}}fl-menu-circle-left{{/if}} hidden">
```

Comment:
Same concern as https://github.com/Fliplet/fliplet-menu-default/pull/25#discussion_r583045668

Reply:
@tonytlwu in this menu transition is working because we use [setTimeout](https://github.com/Fliplet/fliplet-menu-circle/pull/24/files#diff-82df7b14712db47ddbc76d0b8bc9ff3977cc8e1c2217029413661176afc37f97R69) here to give time for the component to remove the `display: none`
[screencast](https://share.getcloudapp.com/7Kupj9ER)

Reply:
Makes sense. Thanks for the explanation and screencast.